src/Fixer/Import/FullyQualifiedStrictTypesFixer.php

```diff
@@ -86,7 +97,7 @@ public function getPriority(): int
 
     public function isCandidate(Tokens $tokens): bool
     {
-        return $tokens->isTokenKindFound(T_FUNCTION);
+        return $tokens->isAnyTokenKindsFound([T_FUNCTION, T_IMPLEMENTS, T_EXTENDS, T_CATCH, T_DOUBLE_COLON]);
```

Comment:
`use trait` should be also supported, it is about the same category

Reply:
I'll look at this but if it requires significant amount of work, I would rather focus on currently supported scenarios and provide trait support in separate PR, to separate authorship.

Reply:
Will be handled in #7568.